package.json

```diff
@@ -51,5 +51,8 @@
     "@types/react": "18.3.1",
     "@types/react-dom": "18.3.0"
   },
-  "packageManager": "yarn@3.5.0"
+  "packageManager": "yarn@3.5.0",
+  "dependencies": {
+    "@iconscout/react-unicons": "~2.0.2"
```

Comment:
I realize this was added by mistake, but let's be more attentive to detail when submitting work to a review.

Reply:
Please keep code reviews polite - this is not the way to make change suggestions.